<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>658</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>658</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>658</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Учет личного  состава подраздения</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Дежурный по академии</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>961</width>
        <height>461</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Номер курса</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество л/с  по списку</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество л/с на лицо</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Причина отсутсвтия</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>271</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Поиск записи:</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите номер подраздения:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>470</y>
        <width>331</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Вывод в Excel:</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите номер подраздения:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Вывести</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Помощник дежурного по академии</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>381</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить запись:</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер подразделения:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2023</year>
         <month>5</month>
         <day>25</day>
        </datetime>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество людей по списку:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество людей на лицо:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Номер курса</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество л/с  по списку</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество л/с на лицо</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Причина отсутсвтия</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Дежурный фельдшер</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить запись:</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер подразделения:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2023</year>
         <month>5</month>
         <day>25</day>
        </datetime>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Список больных(в скобках указать место лечения):</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>361</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Номер курса</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество л/с  по списку</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество л/с на лицо</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Причина отсутсвтия</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>981</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Подключиться к серверу:</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IP-адрес:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>25</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>5555</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
